src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -612,18 +625,20 @@ public function partialRefundAction(int $orderId, Request $request)
      * @return RedirectResponse
      */
     #[AdminSecurity("is_granted('update', request.get('_legacy_controller')) && is_granted('delete', request.get('_legacy_controller'))")]
-    public function standardRefundAction(int $orderId, Request $request)
-    {
-        $formBuilder = $this->get('prestashop.core.form.identifiable_object.builder.cancel_product_form_builder');
-        $formHandler = $this->get('prestashop.core.form.identifiable_object.standard_refund_form_handler');
+    public function standardRefundAction(
+        int $orderId,
+        Request $request,
+        #[Autowire(service: 'prestashop.core.form.identifiable_object.builder.cancel_product_form_builder')] FormBuilderInterface $formBuilder,
+        #[Autowire(service: 'prestashop.core.form.identifiable_object.partial_refund_form_handler')] FormHandlerInterface $formHandler,
```

Comment:
```suggestion
        #[Autowire(service: 'prestashop.core.form.identifiable_object.standard_refund_form_handler')] FormHandlerInterface $formHandler,
```
Same issue that caused the tests to fail and blocked you for weeks, except this one is probably not tested so it was not detected

Please be careful with your modifications and always review your own code!! I warned you to check your whole PR for this kind of mistakes, seems like you didn't take the time to do it...